tests/publishers/fixtures/email-ext004.yaml

```diff
@@ -0,0 +1,37 @@
+publishers:
+  - email-ext:
+      recipients: foo@example.com, bar@example.com
+      reply-to: foo@example.com
+      from: lorem@ipsum.dolor
+      content-type: html
+      subject: Subject for Build ${BUILD_NUMBER}
+      body: The build has finished
+      attach-build-log: false
+      compress-log: false
+      attachments: "*/foo*.log"
+      always: true
+      unstable: true
+      first-failure: true
+      first-unstable: true
+      not-built: true
+      aborted: true
+      regression: true
+      failure: true
+      second-failure: true
+      improvement: true
+      still-failing: true
+      success: true
+      fixed: true
+      fixed-unhealthy: true
+      still-unstable: true
+      pre-build: true
+      matrix-trigger: only-configurations
+      presend-script: "cancel=true"
+      postsend-script: "cancel=true"
+      save-output: true
+      send-to:
+        - developers
+        - requester
+        - culprits
+        - recipients
+        - upstream-committers
```

Comment:
It's worth adding:
* failing-test-suspects-recipients
* first-failing-build-suspects-recipients